ipt/src/mock.rs

```diff
@@ -0,0 +1,98 @@
+//! Mocks for the gradually-update module.
+
+#![cfg(test)]
```

Comment:
You can remove these now, the whole file will skip compilation when tests arent running